api/base/serializers.py

```diff
@@ -274,11 +275,19 @@ class RelationshipField(ser.HyperlinkedIdentityField):
         related_view_kwargs={'node_id': '<_id>', 'wiki_id': '<wiki_pages_current.home>'}
     )
 
+    Field can include optional filters:
```

Comment:
`RelationshipField` formats a link with the view and view_kwargs that get passed to it. The `filter` param will append a filter query to that link. 

The `filter_key` param in develop is essentially the `source_field`. It looks like when you try to filter on a `RelationshipField` the source is `*` so it has to be defined: https://github.com/CenterForOpenScience/osf.io/commit/64fbe9d73d1fb1c7347b58ab29af4f0760a798ba